app/views/spree/components/buttons/_primary.html.erb

```diff
@@ -1,13 +1,15 @@
 <%
-  base_class = "button".freeze
+  base_class = "button-primary".freeze
 
   # Optional props
   disabled = false if local_assigns[:disabled].nil?
+  full_width = false if local_assigns[:full_width].nil?
```

Comment:
I think we should use:

```ruby
full_width = false if local_assigns[:full_width].blank?
```

because as is, it returns `true` even when a `full_width: false` is passed to the partial.

Reply:
I'm not sure I follow. Isn't the logic here: 
- if a value is not provided, use `false`
- if a value is provided, use the value itself

so that if we pass `false`, `full_width` will still be false.

Reply:
if we pass `false`, with current code the line is not executed at full_width is not set. To accomplish that logic you have to use [`.presence`](https://apidock.com/rails/Object/presence)